src/client/components/app/Transactions/WithdrawCreditTx.tsx

```diff
@@ -35,10 +36,12 @@ export const WithdrawCreditTx: FC<BorrowCreditProps> = (props) => {
   const [transactionCompleted, setTransactionCompleted] = useState(0);
   const [transactionLoading, setLoading] = useState(false);
   const [targetAmount, setTargetAmount] = useState('1');
+  const [errors, setErrors] = useState<string[]>(['']);
```

Comment:
@kibagateaux do we want to store errors in the global state?

Reply:
wouldn't it make sense to store errors where the errors are occurring? 

Since we'd want to display them immediately within the same component.